data_struct.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <string.h>
+
+struct Car_type
+
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
+
+struct Car_type2
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
+
+int main(void)
+{
+struct Car_type Chevy;
+struct Car_type2 Ford;
+Chevy.years = 2016;
+Ford.years = 2013;
+strcpy(Chevy.branch, "Chevrolet");
+strcpy(Ford.branch, "Ford");
+strcpy(Chevy.model, "Cruze");
+strcpy(Ford.model, "Fusion");
+
+printf("The branch car 1 is : %s, model %s, with the manufaturing year %d\n", Chevy.branch,Chevy.model, Chevy.years);
+printf("The branch car 2 is : %s, model %s, with the manufaturing year %d\n", Ford.branch, Ford.model, Ford.years);
```

Comment:
How about writing a function to handle printing so that the `printf` code isn't duplicated? This will also make sure that both car variables have the same type (see https://github.com/Jenerth22Rodriguez/C-Book-Exercises/pull/3/files#r999441772).